CONTRIBUTING.md

```diff
@@ -1,45 +1,93 @@
 
-```shell 
-$ pytest --cov=milvus/client --cov-report=html
-```
+If you want to request more features for PyMilvus, you can:
+- file an issue to describe the new features and explain why
+- provide implementation scheme and test scheme (optional)
+- create a PR to realize the new features (optional)
 
-You may need a milvus server which is running when you run unit test. See more details on [Milvus server](https://github.com/milvus-io/milvus).
+If you are interested in existing PRs, you can:
+- join the code review and offer advice
+- guide newcomers to finish the PR process
 
+Note: the problems, features, and questions mentioned here are not limited to Python code. They also refer to all kinds of documents (technical documents, API references, contributing guide, etc.)
 
-## Update CHANGLOG.md
+## PyMilvus Code Structure
+`docs/`: Contains source codes for documentation except for API Reference and uses sphinx to create documentation. Most documentation are in folder `rst` under `docs/source`.
 
-Add issue tips into CHANGLOG.md, make sure all issue tips are sorted by issue number in ascending order.
+`examples/`: Contains python scripts, which can be run directly, to introduce the usage of PyMiluvs API through examples.
 
+`pymilvus/`：PyMilvus source codes.
 
+`tests/`: Unit Test.
 
+`CONTRIBUTING.md`: Contributing guidelines.
+
+`CONTRIBUTING_CN.md`: Contributing guidelines in Chinese.
+
+`LICENSE`: Open Source License that PyMilvus follows.
+
+`Makefile`: Scripts for Github action.
+
+`OWNERS`: This file designates reviewers and approvers for the current directory. They are chosen according to their activity and code contribution. Active contributors are listed as reviewers, undertaking the responsibility of code reviews. Reviewers who have been active and reviewing codes for a period of time are listed as approvers. They are in charge of reviewing content apart from codes. If you create a PR and do not know who to review, you can assign reviewers and approvers from this file to review your PR.
+
+`README.md`: Readme.
+
+`requirements.txt`: Dependencies for developing PyMilvus.
+
+`setup.py`: Package script for PyMilvus.
+
+## Need-to-know before contribution
+
+### PyMilvus Github Workflow
+
+## Need-to-know before PR
+
+### How to write a commit message?
+
+### Sign DCO
+
+## Merge PR
+
+### What will reviewers focus on in reviewing PR?
+
+### What will approvers focus on in reviewing PR?
+
+### Pass all Github Actions
+
+## Congratulations! You are now the contributor to the Milvus community!
+
+Apart from dealing with codes and machines, you can always communicate with people from the Milvus community. Many new faces join in every day, and if you have encountered the same challenges as they are now facing, feel free to help them. You can pass on the collaborative spirit from the helps you acquired when you first join the community. Let us build a collaborative, open-source, open, and tolerant community together!
```

Comment:
Apart from dealing with codes and machines, you are always welcome to communicate with any member from the Milvus community. New faces join us every day, and they may as well encounter the same challenges as you faced beore. Feel free to help them. You can pass on the collaborative spirit from the assistance you acquired when you first joined the community. Let us build a collaborative, open-source, exuberant, and tolerant community together!